sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0-b4 (2021-04-13)
```

Comment:
The CI actually caught this!
```suggestion
## 1.0.0b4 (2021-04-13)

```

Reply:
I didn't even know this was a restriction! Good CI 👍🏻